src/main/resources/view/TaskCard.fxml

```diff
@@ -23,7 +23,7 @@
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="description" text="\$first" styleClass="cell_big_label" />
+        <Label fx:id="description" text="\$first" styleClass="cell_big_label" maxHeight="50" wrapText="true" />
```

Comment:
I'm not sure if `maxHeight` is necessary. I think it causes it to stop wrapping when the max height is reached.

Without `maxHeight`:
![image](https://user-images.githubusercontent.com/65202977/139915753-a87cde71-5d7d-443a-b093-f885eeb8aed1.png)

With `maxHeight`:
![image](https://user-images.githubusercontent.com/65202977/139916624-cd8b5f49-58db-47b3-8054-6eac118d1502.png)

Reply:
Yes @jayasting98, I thought we are going to limit the Task's name to two lines, should I make it four lines? @wonyeji

Reply:
@asherhy @thanwinnikki